src/Wizard.js

```diff
@@ -120,6 +120,13 @@ export default class Wizard extends Webform {
     }));
   }
 
+  beforeNext() {
+    if (this.options.hooks && this.options.hooks.hasOwnProperty('beforeNext')) {
+      return this.hook('beforeNext', this.currentPage, this.submission);
```

Comment:
This will need to wait for a callback from the hook (you can use the last parameter of the hook for a callback) and then call super.beforeNext(). We can't short circuit the calls.